modules/store/testing/src/mock_store.ts

```diff
@@ -103,7 +103,7 @@ export class MockStore<T = object> extends Store<T> {
   }
 
   select(selector: any, prop?: any) {
-    if (typeof selector === 'string' && this.selectors.has(selector)) {
+    if (this.selectors.has(selector)) {
```

Comment:
Can you add a test for the changed behavior?
It's unclear for me which extra cases this change adds, that weren't supported previously.

Reply:
Such a case should have been covered by `modules/store/testing/spec/mock_store.spec.ts:162` (vs 152) in the test    `it('should allow mocking of store.select with a memoized selector using overrideSelector', () => {` however when i attempt to run this test as explained by the docs 
```
yarn && yarn jest modules/store/testing/spec/mock_store.spec.ts --watch
```

 i get.

```
ts-jest[versions] (WARN) Version 4.0.5 of typescript installed has not been tested with ts-jest. If you're experiencing issues, consider using a supported version (>=2.7.0 <4.0.0). Please do not report issues in ts-jest if you are using unsupported versions.
 FAIL  modules/store/testing/spec/mock_store.spec.ts
  ● Test suite failed to run

    TypeScript diagnostics (customize using `[jest-config].globals.ts-jest.diagnostics` option):
    modules/store/testing/spec/mock_store.spec.ts:314:33 - error TS7006: Parameter 'done' implicitly has an 'any' type.

[...more...]
```

Reply:
Without digging too much into your build pipeline I'm wondering if these tests are getting run. The circle build for this pr 
https://app.circleci.com/pipelines/github/ngrx/platform/749/workflows/54e1e0a8-e3ed-4108-a4fa-fda9d5c5d636/jobs/31234

Includes 
```
#!/bin/bash -eo pipefail
yarn nx affected:test --base=$(cat ~/project/master.txt) --head=$CIRCLE_SHA1 --parallel
yarn run v1.17.3
$ nx affected:test --base=17571e5b --head=c4a3d5021f58bc73bc964620d061fe3b27225378 --parallel

>  NX  No projects with "test" were run


———————————————————————————————————————————————

>  NX   SUCCESS  Running target "test" succeeded


Done in 3.09s.
```

This looks offhand like no tests were actually run, that *with* the source code file edit to `modules/store/testing/src/mock_store.ts`